dio/lib/src/multipart_file.dart

```diff
@@ -18,16 +18,56 @@ class MultipartFile {
   ///
   /// [contentType] currently defaults to `application/octet-stream`, but in the
   /// future may be inferred from [filename].
+  @Deprecated(
+    'MultipartFile.clone() will not work when the stream is provided, use the MultipartFile.fromStream instead.'
+    'This will be removed in 6.0.0',
+  )
   MultipartFile(
-    Stream<List<int>> stream,
+    Stream<List<int>>? stream,
```

Comment:
Is it nullable for any special concerns?

Reply:
No this is a mistake from the refactor.

Reply:
Fixed

Reply:
Fixed

Reply:
Forgot to run push or something?

Reply:
Yea... pushed to wrong remote :)